src/components/Accordion/Accordion.module.css

```diff
@@ -0,0 +1,75 @@
+.sbui-accordion-container {
+  @apply flex flex-col rounded-md;
+}
+
+.sbui-accordion-container--bordered {
+  @apply bg-white dark:bg-dark-700;
+  border: 1px solid #e0e0e0;
```

Comment:
might be possible to use:
```css
.sbui-accordion-container--bordered {
  @apply bg-white dark:bg-dark-700;
  @apply border dark:border-dark
}
```

Reply:
Ah thanks yeah sorted, it was `dark:border-darkmode` rather than just `dark:border-dark`. Fixed a couple of similar bits too.